index.js

```diff
@@ -82,17 +86,22 @@ Decorator.prototype.getTile = function(z, x, y, callback) {
     });
 };
 
-function loadAttributes(keys, client, cache, callback) {
+function loadAttributes(useHashes, keys, client, cache, callback) {
     // Grab cached values from LRU, leave
     // remaining for retrieval from redis.
     var replies = [];
     var loadKeys = [];
     var loadPos = [];
+    var multi = client.multi();
+
     for (var i = 0; i < keys.length; i++) {
         var cached = cache.get(keys[i]);
+
         if (cached) {
             replies[i] = cached;
         } else {
+            if (useHashes) multi.hgetall(keys[i]);
```

Comment:
Mind adding brackets? These always feel like running around in the streets without shoes on - just waiting to across some glass.